eng/Signing.props

```diff
@@ -0,0 +1,24 @@
+<Project>
+
+  <ItemGroup>
+    <FileSignInfo Include="mlaunch.exe" CertificateName="3PartySHA2" />
+    <FileSignInfo Include="System.Net.Http.dll" CertificateName="3PartySHA2" />
+    <FileSignInfo Include="System.Numerics.dll" CertificateName="3PartySHA2" />
+    <FileSignInfo Include="System.Xml.dll" CertificateName="3PartySHA2" />
+    <FileSignInfo Include="Xamarin.Hosting.dll" CertificateName="3PartySHA2" />
+    <FileSignInfo Include="Xamarin.Mac.dll" CertificateName="3PartySHA2" />
+    <FileSignInfo Include="Mono.Security.dll" CertificateName="3PartySHA2" />
+    <FileSignInfo Include="mscorlib.dll" CertificateName="3PartySHA2" />
+    <FileSignInfo Include="System.Core.dll" CertificateName="3PartySHA2" />
+    <FileSignInfo Include="System.dll" CertificateName="3PartySHA2" />
```

Comment:
@MattGal I didn't find any other to list files using some path pattern matching. I checked [other dotnet repos](https://github.com/search?q=org%3Adotnet+filename%3ASigning.props&type=Code) and it seemed they all also do this. I tried to do `<ItemsToSign Remove=...` as well but no luck there either.

However, I am not sure it is 100% safe to have `System.dll` here..